governance/program/tests/program_test/cookies.rs

```diff
@@ -160,6 +160,12 @@ pub struct SignatoryRecordCookie {
     pub signatory: Keypair,
 }
 
+#[derive(Debug)]
+pub struct SignatoryRecordCookieWithoutKeypair {
```

Comment:
nit: Could we reuse `SignatoryRecordCookie` with `signatory`: `Keypair` as `Option<Keypair> ` or maybe just use `SignatoryRecordCookie` as is?

Reply:
resolved